explorer/client/src/components/Table.tsx

```diff
@@ -33,16 +33,21 @@ const Loading = ({ colCount, msg }: LoadingProps) => {
   )
 }
 
-interface EmptyProps {
-  colCount: number
-  msg?: string
+const Empty: React.FC<MsgProps> = ({ colCount, msg }) => {
+  return (
+    <TableRow>
+      <MuiTableCell component="th" scope="row" colSpan={colCount}>
+        {msg || 'No results'}
```

Comment:
```suggestion
        {msg ?? 'No results'}
```
Good time to start using nullish coalescing :)